daemon/defaults/defaults.go

```diff
@@ -19,7 +19,7 @@ const (
 	RuntimePath = "/var/run/cilium"
 
 	// RuntimePathRights are the default access rights of the RuntimePath directory
-	RuntimePathRights = 0770
+	RuntimePathRights = 0775
```

Comment:
Otherwise I have to change the directory group to belong to `ciliium` as well.

I couldn't reach the cilium socket inside without `775`